sale_coupon_mass_mailing/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This addon adds a smart-button in promotions called "Mailings" that allows you to link mass-mailing.
+In the event that there is no mass-mailing created, when the smart-button is clicked, one will be created with the contacts previously defined in the promotion, otherwise the existing mailings will be listed.
```

Comment:
```suggestion
In the case that there is no mass-mailing created yet, when clicking the smart-button, one will be created with the contact domain previously defined in the promotion, otherwise, the existing mailings will be listed.
```